test/bats/aws.bats

```diff
@@ -81,7 +81,7 @@ teardown_file() {
    [[ "${result//$'\r'}" == "BeforeRotation" ]]
 
    aws ssm put-parameter --name $PM_ROTATION_TEST_NAME --value AfterRotation --type SecureString --overwrite --region $REGION
-   sleep 40
+   sleep 180
```

Comment:
why do we need to increase the sleep time? (applies to all increased sleep duration)

Reply:
Minimum time period between 2 `nodepublish` volume is 1 minute + some jitter (generally 10 sec).

changing it to 120 sec.

Reply:
Do we want to leave this as a `# comment`?  Seems helpful.

Reply:
Thanks, Done.